src/components/Users/Users.js

```diff
@@ -89,8 +56,26 @@ const Users = (props) => {
       </div>
       <div>
         {user.followed
-          ? <button disabled={followingInProgress.some((id) => id === user.id)} type="submit" onClick={() => goUnfollow(user)}>Unfollow</button>
-          : <button disabled={followingInProgress.some((id) => id === user.id)} type="submit" onClick={() => goFollow(user)}>Follow</button>}
+          ? (
+            <button
+              type="submit"
```

Comment:
Consider creating a separate function that returns these buttons JSX and accepts onclick handler and a text.

Reply:
/fxd